src/main/java/ee/tuleva/onboarding/common/FtpClient.java

```diff
@@ -0,0 +1,56 @@
+package ee.tuleva.onboarding.common;
+
+import lombok.*;
+import org.apache.commons.net.*;
+import org.apache.commons.net.ftp.*;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.*;
+
+@RequiredArgsConstructor
+public class FtpClient {
+    private final String server;
+    private final String user;
+    private final String password;
+    private final int port;
+
+    private FTPClient ftp;
+
+    public void open() throws IOException {
```

Comment:
I would eliminate the checked **IOExceptions** from the method signatures. Rather catch it and wrap it with the [UncheckedIOException](https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html).